### ISSUE TITLE
Failed to build via jitpack.io

### ISSUE THREAD

**Issue by Sufaev** [WorldWindEarth/WorldWindJava#48]
Hi.

Is it possible to fix build scripts to be able to import develop-SNAPSHOT from github via jitpack.io?

Now it shows following errors:
https://jitpack.io/com/github/WorldWindEarth/WorldWindJava/develop-a840f19ad8-1/build.log

**Comment by PJHogan** [comment]
Seems dated but doable,
https://www.jitpack.io/
https://github.com/jitpack-io/jitpack.io

**Comment by wcmatthysen** [comment]
@Sufaev, we can look at [JitPack](https://jitpack.io) as one way to distribute snapshots. It seems like an interesting alternative to hosting artifacts on Bintray. To address your issue: I see from the error message in the JitPack build:

    * What went wrong:
    A problem occurred evaluating root project 'worldwind'.
    > Could not set unknown property 'archiveBaseName' for task ':extensionsJar' of type 
    org.gradle.api.tasks.bundling.Jar.

It looks like one of the issues that I fixed in pull-request #44 for the Gradle config. Specifically, if you look at this commit: https://github.com/WorldWindEarth/WorldWindJava/pull/44/commits/ec9415a26cadfddae0f33b1c3b123257db0a61cf you'll see I changed `archiveBaseName` to `baseName`. I think the issue will be solved once we merge those changes into `develop`.
